CHANGELOG.md

```diff
@@ -1,5 +1,89 @@
 # Change Log
 
+## [1.1.0](https://github.com/stellargraph/stellargraph/tree/v1.1.0)
+
+[Full Changelog](https://github.com/stellargraph/stellargraph/compare/v1.0.0...v1.1.0)
+
+Jump in to this release, with the new and improved [demos and examples][demos-1.1.0]:
+
+- Neo4j graph database support: [Cluster-GCN][neo4j-cluster-1.1.0], [GraphSAGE][neo4j-gs-1.1.0], [all demos][neo4j-demos-1.1.0]
+- [Semi-supervised node classification via GCN, Deep Graph Infomax and fine-tuning][dgi-fine-tuning-1.1.0]
+- [Loading data into StellarGraph from NumPy][loading-numpy-1.1.0]
+- [Link prediction with Metapath2Vec][metapath-lp-1.1.0]
+- [Unsupervised graph classification/representation learning via distances][unsup-graph-rl-1.1.0]
+- [RGCN section of Node representation learning with Deep Graph Infomax][unsup-rgcn-1.1.0]
+- Node2Vec with StellarGraph components: [representation learning][keras-n2v-rl-1.1.0], [node classification][keras-n2v-nc-1.1.0]
+- Expanded Attri2Vec explanation: [representation learning][a2v-rl-1.1.0], [node classification][a2v-nc-1.1.0], [link prediction][a2v-lp-1.1.0]
+
+[demos-1.1.0]: https://stellargraph.readthedocs.io/en/v1.1.0/demos/index.html
+[neo4j-demos-1.1.0]: https://stellargraph.readthedocs.io/en/v1.1.0/demos/connector/neo4j/index.html
+[neo4j-cluster-1.1.0]: https://stellargraph.readthedocs.io/en/v1.1.0/demos/connector/neo4j/cluster-gcn-on-cora-neo4j-example.html
+[neo4j-gs-1.1.0]: https://stellargraph.readthedocs.io/en/v1.1.0/demos/connector/neo4j/undirected-graphsage-on-cora-neo4j-example.html
+[dgi-fine-tuning-1.1.0]: https://stellargraph.readthedocs.io/en/v1.1.0/demos/node-classification/gcn-deep-graph-infomax-fine-tuning-node-classification.html
+[loading-numpy-1.1.0]: https://stellargraph.readthedocs.io/en/v1.1.0/demos/basics/loading-numpy.html
+[metapath-lp-1.1.0]: https://stellargraph.readthedocs.io/en/v1.1.0/demos/link-prediction/metapath2vec-link-prediction.html
+[unsup-graph-rl-1.1.0]: https://stellargraph.readthedocs.io/en/v1.1.0/demos/embeddings/gcn-unsupervised-graph-embeddings.html
+[unsup-rgcn-1.1.0]: https://stellargraph.readthedocs.io/en/v1.1.0/demos/embeddings/deep-graph-infomax-embeddings.html#Heteogeneous-models
+[keras-n2v-rl-1.1.0]: https://stellargraph.readthedocs.io/en/v1.1.0/demos/embeddings/keras-node2vec-embeddings.html
+[keras-n2v-nc-1.1.0]: https://stellargraph.readthedocs.io/en/v1.1.0/demos/node-classification/keras-node2vec-node-classification.html
+[a2v-rl-1.1.0]: https://stellargraph.readthedocs.io/en/v1.1.0/demos/embeddings/attri2vec-embeddings.html
+[a2v-nc-1.1.0]: https://stellargraph.readthedocs.io/en/v1.1.0/demos/node-classification/attri2vec-node-classification.html
+[a2v-lp-1.1.0]: https://stellargraph.readthedocs.io/en/v1.1.0/demos/link-prediction/attri2vec-link-prediction.html
+
+### Major features and improvements
+
+- Support for the Neo4j graph database has been much improved:
```

Comment:
possibly a bit colloquial?  simple fix below, but might be an even better wording?
```suggestion
- Support for the Neo4j graph database has been significantly improved:
```